composer.json

```diff
@@ -167,6 +167,7 @@
     },
     "conflict": {
         "symfony/dependency-injection": "3.4.15|3.4.16",
+        "symfony/monolog-bundle": "3.5.0",
```

Comment:
Well every newer version of monolog-bundle will be in conflict, we should lock version in last 3.4

Reply:
+ this branch should be build upon 7.3 as this problem will occur even there

Reply:
Maybe they just did something wrong with their dependecies. It does not make any sence to me that monolog v2 requires symfony 5 which is in beta since yeasterday.

Reply:
In 7.3 it is not a problem, I guess there are some older dependencies which blocks upgrading.

Anyway I'm ok to fix `symfony/monolog-bundle` to 3.4